scripts/setup_ubuntu.sh

```diff
@@ -96,9 +96,9 @@ GIT=git
 LIBRARIES="libxml2-dev llvm-10-dev"
 
 apt install -yq --allow-downgrades \
-  $DEVELOPER_TOOLS \
+  "$DEVELOPER_TOOLS" \
```

Comment:
This is supposed to be word split. Either use an array or remove the quotes. Same with `LIBRARIES` below.